proxy.conf.nexus.template

```diff
@@ -43,6 +43,13 @@ server {
 #        send_timeout       300;
     }
 
+    location /{PHPLDAPADMIN_URI} {
+	proxy_pass    http://{PHPLDAPADMIN_URI};
+        proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
```

Comment:
Looking at the actual file there doesn't seem to be spacing issues but if there is just remove that extra space on this line